doc/glossary.rst

```diff
@@ -381,17 +377,11 @@ general neuroimaging concepts. If you think a term is missing, please consider
 
     whitening
         A linear operation that transforms data with a known covariance
-        structure into "whitened data" which has a covariance structure that
-        is the identity matrix. In other words it creates virtual channels that
+        structure into "whitened data", which has a covariance structure equal to
+        the identity matrix. In other words, whitening creates virtual channels that
         are uncorrelated and have unit variance. This is also known as a
         sphering transformation.
 
-        The term "whitening" comes from the fact that light with a flat
-        frequency spectrum in the visible range is white, whereas
-        non-uniform frequency spectra lead to perception of different colors
-        (e.g., "pink noise" has a ``1/f`` characteristic, which for visible
-        light would appear pink).
```

Comment:
This was added intentionally, please put it back (see e.g. the discussion in https://github.com/scipy/scipy/issues/12446#issuecomment-683454152 / https://github.com/mne-tools/mne-python/issues/7425#issuecomment-679324485)

Reply:
Ah, thanks for the links, I'd never have thought that this could be offensive. Sure, I'll put it back.